components/automate-chef-io/content/docs/ldap.md

```diff
@@ -1,82 +1,308 @@
+policies will be incorrectly labelled as `ldap`.
+
+{{< info >}}
+Users who log in via SAML will have a session time of 24 hours before needing to log in again.
+Local, MSAD, and LDAP users will have their Chef Automate sessions refreshed while they maintain an active
+browsing session of the Chef Automate UI or until they log out directly.
+{{< /info >}}
+
 ## Overview
 
-This is documentation for using Chef Automate's LDAP and Microsoft Active Directory integrations.
-First, we detail how the login process works when logging in via LDAP.  Then,
-we describe common directory layouts, and what Chef Automate config would be
-used for them. At the end, you will find a troubleshooting section collecting
-common cases.
+This is documentation for configuring Chef Automate's Lightweight Directory Application Protocol (LDAP) and Microsoft Active Directory (MSAD) integrations. LDAP is an established and open standard protocol for interacting with directory servers. A directory server stores information--in this case information for authenticating and authorizing users--in a tree of entries. (It is not a relational database.)
 
-### Login with LDAP
+## Microsoft Active Directory
 
-Once the user has provided a username and password at the login screen, Chef
-Automate goes through a sequence of operations to complete the login: Connect,
-Bind, User Search, Login Bind, and Group Search.
+Microsoft Active Directory (MSAD) is a type of directory server that supports LDAP. Chef Automate comes with a default LDAP configuration for MSAD.
+The Chef Automate default MSAD configuration is a minimal configuration for standard MSAD systems which you can extend by overriding default values and using additional configuration options.
+Chef Automate's default configuration for Microsoft AD is specific to LDAP.
+To configure Microsoft AD using SAML, see the [SAML documentation]({{< relref "saml.md" >}}).
 
-### Authorization with LDAP
+## Changing Chef Automate Configuration
 
-Chef Automate supports using your LDAP server to authenticate users and define their permissions based on [LDAP group memberships]({{< ref "authorization-overview.md" >}}).
+If you need to change your configured external identity provider settings, replace your existing configuration by following these steps:
 
-#### Connect
+1. Run `chef-automate config show config.toml`.
+2. Edit `config.toml` to replace the `dex.sys.connectors` section with the configuration values for your new identity provider.
+3. Run `chef-automate config set config.toml` to set your updated configuration.
+
+### Minimal MSAD Configuration
+
+base_user_search_dn
+: "your base user search DN"
+
+base_group_search_dn
+: "your base group search DN"
+
+bind_dn
+: "your bind_dn"
+
+bind_password
+: "your bind_password"
+
+ca_contents
+: Your certificate authority (CA) certificate contents. You can provide multiple PEM-encoded CA certs. Optional.
```

Comment:
I do not find where L65-L78 come from in the original; also, they seem to really be already covered by L96-101 so consider deleting L65-L78.